stingray/stats.py

```diff
@@ -776,3 +786,332 @@ def _pavnosigfun(power, nspec):
         m -= 1
 
     return sum
+
+
+def power_confidence_limits(preal, n=1, alpha=0.16):
+    """Confidence limits on power, given a signal power in the PDS/Z search.
+
+    Adapted from Vaughan et al. 1994, noting that, after appropriate
+    normalization of the spectral stats, the distribution of powers in the PDS
+    and the Z^2_n searches is always described by a noncentral chi squared
+    distribution.
+
+    Parameters
+    ----------
+    preal: float
+        The underlying, signal-generated value of power
```

Comment:
I don't really understand what this means. Is this the observed value of the power? Based on this description, I would assume that it isn't, but rather the unobserved "true" power spectrum? But that isn't observed, so how can it be an input parameter? Help?